src/components/sync-configurator/BaseSyncConfigurator.jsx

```diff
@@ -22,8 +36,19 @@ export default class BaseSyncConfigurator extends React.Component {
         },
     ];
 
+    selectionChange = selection => {
+        this.setState({ selection });
+    };
+
+    startSynchronization = () => {
+        // TODO: Render new dialog to show summary and select destination instances
+        queryMetadata(this.props.d2, "organisationUnit", "zGv4gjlRmCX").then(result =>
```

Comment:
[DEBUG]: Should open the new synchronization dialog to show a summary of sync and select instances